app/src/main/res/values/settings_keys.xml

```diff
@@ -55,6 +55,25 @@
         <item>30000</item>
     </string-array>
 
+    <string name="progressive_load_interval_key">progressive_load_interval</string>
+    <string name="progressive_load_interval_bytes_default_value">16384</string>
+    <string-array name="progressive_load_interval_descriptions">
+        <item>1 KiB</item>
```

Comment:
Keeping this at 1024/KiB for now since that is the unit used in ExoPlayer. Will test *1000 on my side and see if there is anything noticeable before making the change.

Reply:
> Otherwise the PR LGTM

Still interested in this change? Base on my own tests, there seems to be little to no difference between 1000x vs 1024x, though I'd still go for what exo devs are using.